python/core/model_processing/converters/onnx_converter.py

```
@@ -920,6 +920,10 @@ def quantize_layer(
             onnx.NodeProto:
                 A quantized node or list of nodes replacing the initial node.
         """
+        try:
+            opset_version = model.opset_import[0].version
+        except Exception:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)